pkg/resource/pod_distruption_budget_test.go

```diff
@@ -0,0 +1,71 @@
+package resource_test
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/cockroachdb/cockroach-operator/pkg/labels"
+	"github.com/cockroachdb/cockroach-operator/pkg/resource"
+	"github.com/cockroachdb/cockroach-operator/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	policy "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func TestPDBBuilder(t *testing.T) {
+
+	var MaxInt int32 = 1
```

Comment:
`maxUnavailable`? Otherwise I'm expecting some limit on the integer, although it can be my personal perception.

Reply:
changed